v20.2/performance-benchmarking-with-tpc-c-10k-warehouses.md

```diff
@@ -32,7 +38,7 @@ TPC-C provides the most realistic and objective measure for OLTP performance at
 
 ### Request a trial license
 
-Reproducing CockroachDB's 10,000 warehouse TPC-C results involves using CockroachDB's [partitioning](partitioning.html) feature to ensure replicas for any given section of data are located on the same nodes that will be queried by the load generator for that section of data. Partitioning helps distribute the workload evenly across the cluster.
+Reproducing CockroachDB's 13,000 warehouse TPC-C results involves using CockroachDB's [partitioning](partitioning.html) feature to ensure replicas for any given section of data are located on the same nodes that will be queried by the load generator for that section of data. Partitioning helps distribute the workload evenly across the cluster.
```

Comment:
I'm inclined to edit this to be less specific so we don't have to change it again.  And on the other pages too, if it's also there. Something like

> Replicating *these* TPC-C results ...

Any objections?